x-pack/test/functional/apps/ml/settings/filter_list_creation.ts

```diff
@@ -24,34 +21,30 @@ export default function ({ getService }: FtrProviderContext) {
     after(async () => {
       await ml.api.cleanMlIndices();
       // clean up created filters
-      await asyncForEach(testDataList, async ({ filterId }) => {
-        await ml.api.deleteFilter(filterId);
-      });
+      await ml.api.deleteFilter(filterId);
     });
 
-    it('filter list creation creates new filter list', async () => {
+    it('creates new filter list', async () => {
       await ml.testExecution.logTestStep(
         'filter list creation loads the filter list management page'
       );
       await ml.navigation.navigateToMl();
       await ml.navigation.navigateToSettings();
       await ml.settings.navigateToFilterListsManagement();
 
-      await ml.testExecution.logTestStep('filter list creates new filter lists');
-      for (const testData of testDataList) {
-        await ml.testExecution.logTestStep('filter list creation loads the filter creation page');
-        await ml.settingsFilterList.navigateToFilterListCreationPage();
-
-        await ml.testExecution.logTestStep('filter list sets the list name and description');
-
-        const { filterId, description, keywords } = testData;
-        await ml.settingsFilterList.setFilterListId(filterId);
-        await ml.settingsFilterList.setFilterListDescription(description);
-        await ml.settingsFilterList.addFilterListKeywords(keywords);
-        await ml.testExecution.logTestStep('filter list saves the settings');
-        await ml.settingsFilterList.saveFilterList();
-        await ml.settingsFilterList.assertFilterListRowExists(filterId);
-      }
+      await ml.testExecution.logTestStep('filter list creation creates new filter lists');
```

Comment:
I think this line could be removed as you're already logging more detailed test steps in the next lines.

Reply:
Removed here https://github.com/elastic/kibana/pull/77566/commits/db1bdd7aba231b61e98e3decf5ad481b20b85eaf